### DISCUSSION TITLE
Wiki with documentation

### DISCUSSION THREAD

**Discussion by Hugo-Persson** [ghostfolio/ghostfolio#2818] [Ideas]
Could we open a Wiki for this project so we could start building up some documentation about what different functions work?

**Comment by dtslvr** [comment]
Hi @Hugo-Persson

Thanks for the input, I'm still undecided about your idea.

On the one hand, there is more and more functionality that users may not be aware of. On the other hand, I wonder if it makes sense to start an additional knowledge base that needs to be kept up to date in addition to the [README](https://github.com/ghostfolio/ghostfolio/blob/main/README.md), [GitHub Discussions](https://github.com/ghostfolio/ghostfolio/discussions), [FAQ](https://ghostfol.io/en/faq) and hints directly in the app (my preferred option).

How did you imagine the structure of the wiki?

**Reply by Hugo-Persson** [reply]
Maybe in-app hints better. I was thinking like a collection of how to do common actions. Like how to create manual asset. Or how scraper configuration works. How to add currency, real estate...

**Reply by Marketos-Damigos** [reply]
The readme and faq lack so much information regarding setting up the environment as well the functionality.

**Comment by The-Inamati** [comment]
I would second this. I just started the container and am struggling a lot to understand how to get this running in terms of configuration of the application itself not the container.

**Comment by unarmedpuppy** [comment]
I'm very interested in using this tool, self-hosted it to try it out, but I have absolutely no idea how I'm supposed to use any of the features. I can add cash accounts...but nothing else seem intuitive. Am I missing something obvious here to get started? Like adding my stock portfolios, other holdings, etc. A wiki or any documentation at all for how to use the tool would be helpful. 

`docker-compose down` for now

**Reply by Marketos-Damigos** [reply]
I had the exact same reaction. The website barely gives any basic info and only guides you to the premium version, which still has zero documentation.

**Comment by diamant-x** [comment]
This needs to be high in the list. Currently there is not "get Started" page anywere. FAQ assumes you are already familiar with concepts from ghostfolio, and most of its answers are just too simple. Example: 'How to add other currencies' -> go to control panel, add currency. That's not much useful, How will future exchange rates be updated? Only manually? (Which is the only thing visible.
What are platforms? What is expected to be input in each field. It mentions yahoo somewhere around but even for that there's no example.

[This ](https://docs.firefly-iii.org/how-to/firefly-iii/finances/accounts/)could be an example of structure, but basically there needs to be a clear page defining all concepts in ghostfolio, including what they refer to in "real life" examples, and how they should be used.
And then another with tutorials on use cases "what should you do to track ETFs (from beginning to end)?" etc..

**Comment by colorcube** [comment]
I agree that a documentation is missing.

While it would be nice that the app itself gives more hints how to do things, I can't image how to include all the necessary information into a faq.

I see at least following topics:

- what does it do
- concepts
- usage (ui)
- examples/use cases
- custom setup (scraper config)
- updates

This would include useful information for self-hoster, but also for cloud users.

In my opinion, a faq can't include such information in a good way.

In github discussions I could imagine that the same questions are asked again and again, mostly because many of them don't include the full solution to a problem but only "works for me now".

**Comment by jmrenshaw** [comment]
I'm willing to make a start on some documentation just using a docs folder.  It will help me learn how the app works (from a user perspective) and if it's any good you can merge it in.

**Reply by colorcube** [reply]
Did you find some time to work on documentation? I plan to retry ghostfolio and wonder if we could join forces.

**Reply by jmrenshaw** [reply]
I got as far as forking the repo 😄 , but yes happy to join forces.

**Reply by colorcube** [reply]
After a bit more testing I decided to use another software. So no documentation work from my side. Sorry

**Reply by diamant-x** [reply]
>  I decided to use another software.

Which one? Just to compare the feature set.

**Reply by colorcube** [reply]
https://www.portfolio-performance.info/

**Comment by rosmur** [comment]
A year since this issue has been open and there seems to be no progress. At this point it seems like there's a bunch of marketing about features that basically don't exist.

**Reply by dtslvr** [reply]
Thanks for your feedback. To help us understand what you’re referring to, could you please specify which features you feel are marketed but don’t actually exist?

**Reply by rosmur** [reply]
Socks, crypto, bonds. Everything other than cash can't be entered in.

**Reply by matzingerhh** [reply]
Hi, this is not true. I started the Container a week ago and was able to add a lot of stuff.

**Comment by matzingerhh** [comment]
Hi I think a documentation would be helpful. I don't see a book but some starter would be very helpful. I did a lot by try and error. I learned a lot but I think I would be handier for starters. 
If you decide to create a document I can contribute. Of cause there should be something that avoid that the docu get outdated by changes. 
Regards
Mathias

**Comment by killerra** [comment]
I’ve just started using Ghostfolio, and many parts of the setup process are not very straightforward for new users.

For example, the JSON import flow is not obvious. It took me a while to realize that I needed to open the Accounts page, copy the account ID from the URL, and manually add it to the JSON import data. I also still have not found clear guidance on how to add cryptocurrency holdings from exchanges like Kraken.

I searched with Google and LLM tools, but I still struggled to find clear answers. Better and more structured documentation would make onboarding much easier. Since cryptocurrencies are also listed as a feature on the Ghostfolio website, I would expect this setup to be easier to understand for new users.

I think clearer documentation and onboarding would also help make the product more attractive to potential premium subscribers.
